src/app/widgets/weather/weather-admin/weather-admin.component.html

```diff
@@ -0,0 +1,30 @@
+<form [formGroup]="adminForm" (ngSubmit)="onSubmit()">
+    <div class="weather-admin">
+        <div class="close">
+            <a [routerLink]="['/']"><img src="../assets/x.png" alt=""/></a>
+        </div>
+        <img src="../assets/cloud-sun.png" alt=""/>
+        <label for="country">Write down the Country Name</label>
+            <div class="country">
+                <input type="text" formControlName="country">
+            </div>
+        <label for="city">Check the weather for...</label>
+            <div class="select-city">
+                <select id="city" formControlName="city">
+                    <option value="{{ '' }}"></option>
+                    <option *ngFor="let city of cities; index as i" value="{{ i }}">{{ cities[i].name }}</option>
```

Comment:
you have city already in variable - so instead of cities[i].name, you can use city.name.